### ISSUE TITLE
Utils "get_key_value_from_url" should not be case sensitive

### ISSUE THREAD

**Issue by mperesson** [bukson/steampy#123]
This method allow to get params from URL but is case sensitive.
I had the case when someone send me his trade url like this: 

https://steamcommunity.com/tradeoffer/new/?Partner=123456789&token=XXXXXX

get_key_value_from_url method was not able to retrieve partner value.

Using CaseInsensitiveDict form requests package can be useful:

```
def get_key_value_from_url(url: str, key: str) -> str:
    params = urlparse.urlparse(url).query
    return CaseInsensitiveDict(urlparse.parse_qs(params))[key][0]
```

**Comment by crey4fun** [comment]
https://stackoverflow.com/questions/24699643/are-query-string-keys-case-sensitive As discussed here, in theory it should be case sensitive.
Anyone else thinks the method should be case insensitive?

**Comment by mperesson** [comment]
From a standard URL perspective, it definitely should be case sensitive. 
In the case of steam trade urls, we just want to get partner/token values so it make sens to me. But I maybe should normalize my trade urls before using steampy.

**Comment by crey4fun** [comment]
I don't think the `get_key_value_from_url` method or your own code should be fixed.
I think the `make_offer_with_url` should automatically use the lowercase url.
Thanks for pointing this out, mperesson.

**Comment by mperesson** [comment]
I used `get_key_value_from_url` to do get partner/token for other tasks that's why I pointed it out, but it definitely should be elsewhere

You probably should add a method that normalize steam URL in utils.
This problem should not happen often, but i had 1 user out of ~2500 that send me this kind of url.

**Comment by bukson** [comment]
Fixed in 5e248310563bb9c651695003b6b7eb35ea327889, now one can pass parameter (default True) for case sensitivity.
